company-ycmd.el

```diff
@@ -436,7 +436,7 @@ If CB is non-nil, call it with candidates."
        (ycmd-running?)
        (or (not (company-in-string-or-comment))
            (company-ycmd--in-include))
-       (or (company-grab-symbol-cons "\\.\\|->\\|::" 2)
+       (or (company-grab-symbol-cons "/\\|\\.\\|->\\|::" 2)
```

Comment:
I'm not very familiar with Emacs on Windows, but a quick search indicated that forward slashes are also valid for specifying file paths on Windows Emacs?